airbyte-integrations/connectors/source-github/source_github/schemas/issue_events.json

```diff
@@ -73,6 +90,244 @@
           "$ref": "user.json"
         }
       }
+    },
+    "performed_via_github_app": {
```

Comment:
Nit: should we make a json for this, since we define it twice and it's long?

Reply:
Yes, I thought about it initially 

I have added `"performed_via_github_app"` sub-object to 3 streams: `comments`, `issue_events`, `issues`

But in all 3 streams, this sub-object is different here and there 
I just decided to inline this sub-object to all these 3 streams until we will know some new information.

Reply:
Ah gotcha 👍🏻

Reply:
Are we sure about that? I have compared the field schema in https://docs.github.com/en/enterprise-server@3.6/rest/issues/events, https://docs.github.com/en/enterprise-server@3.6/rest/issues/comments and https://docs.github.com/en/enterprise-server@3.6/rest/issues/issues and they are actually the same. I think this would be relevant to figure this out to avoid other schema changes and prevent loss of data for the customers

Reply:
Actually, all of these PR changes are based on real data. We gathered data from our testing accounts, and then I added updates to the schema to conform to the data.

Here are some details:

Building a good schema is not an easy task.

1. incoming JSON objects from our testing accounts may not cover all possible variations.
2. Fully relying on documentation can also be dangerous. For example, I found some date-time fields that were marked as non-null in the documentation but were actually nullable.
2. GitHub API data has greatly increased over the last year. Even if we build a good schema initially, it could quickly become outdated.

RESUME:  building a good schema is a continuous process, and we shouldn't rely solely on documentation.

P. S. I can actually obtain the "performed_via_github_app" object from the documentation, and there's a high chance that this object is the same across all streams.